api_docs/changelog.md

```diff
@@ -20,6 +20,16 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 8.0
 
+**Feature level 192**
+
+* [`POST /register`](/api/register-queue),
+  [`GET /events`](/api/get-events): Added `create_multiuse_invite_group_id`
+  field in response which represents the `id` of user_group whose members
+  can create invite link to realm.
+
+* `PATCH /realm`: Added a new parameter `create_multiuse_invite_group_id`
+  for changing the user_group whose members can create invite link.
```

Comment:
The writing style here feels a lot more sloppy than existing API changelog entries; please read some similar ones and polish this.